mapper/mapper.go

```diff
@@ -17,6 +17,8 @@ import (
 const (
 	InstanceLabel                = dtwebhook.LabelInstance
 	UpdatedViaDynakubeAnnotation = "dynatrace.com/updated-via-operator"
+	ConflictErrorMessage         = "namespace matches two or more DynaKubes which is unsupported. " +
```

Comment:
should be named `errorConflictingNamespace` or similar to match the other errors